scripts/test-size.js

```diff
@@ -15,42 +14,23 @@ try {
   process.exit(1)
 }
 
-// The last recorded size for these assets, as reported by `yarn build`.
-const LAST_SIZE_MAIN_KB = 435
-
-// This is the async-loaded js, called <number>.<hash>.js, with a matching css file.
-const LAST_SIZE_ENTRY_KB = 1442
-
+// The last recorded size for main.<hash>.js, as reported by `yarn build`.
+const SIZE_MAIN_KB = 1790
 const SIZE_TOLERANCE_KB = 10
 
-const jsEntrypoints = entrypoints.filter((entrypoint) => entrypoint.endsWith('js'))
-assert(jsEntrypoints.length === 3)
```

Comment:
Why we don't want to assert on this anymore?

Reply:
https://github.com/Uniswap/interface/pull/6459#discussion_r1180898182